strapi/strapi
# fix: Removed back button from Role page

What does it do?
Removed the back button from the Roles page.

Why is it needed?
To fix issue [20818](https://github.com/strapi/strapi/issues/20818): The role page has a back button but it shouldn’t 

How to test it?

- Go to U&P plugin > Roles
- See the Back button

Related issue(s)/PR(s)
fixes #20818
